tensorflow-serving/0000-tensorflow_patches.patch

```diff
@@ -25,28 +25,62 @@ index 54d52e2..14a435e 100644
  load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 diff --git a/third_party/tensorflow/BUILD b/third_party/tensorflow/BUILD
 new file mode 100644
-index 0000000..e69de29
+index 00000000..e69de29b
 diff --git a/third_party/tensorflow/tensorflow_patches.patch b/third_party/tensorflow/tensorflow_patches.patch
 new file mode 100644
-index 0000000..0707494
+index 00000000..b4ad792a
 --- /dev/null
 +++ b/third_party/tensorflow/tensorflow_patches.patch
-@@ -0,0 +1,182 @@
-+From 4bc378807dc11e444e4d00ce2a94b0fbe709f5d3 Mon Sep 17 00:00:00 2001
-+From: Deepali Chourasia <deepali@us.ibm.com>
-+Date: Fri, 18 Dec 2020 05:43:20 +0000
+@@ -0,0 +1,293 @@
++From f31d6a6ad09757250f88f331003c07034bc7e019 Mon Sep 17 00:00:00 2001
++From: Nishidha Panpaliya <npanpa23@in.ibm.com>
++Date: Thu, 7 Jan 2021 08:47:01 -0500
 +Subject: [PATCH] TF 2.4 patches
 +
 +---
-+ tensorflow/tensorflow.bzl                          |  4 +-
-+ .../com_google_absl_fix_mac_and_nvcc_build.patch   | 45 +++++++++++++---------
-+ third_party/gpus/find_cuda_config.py               |  2 +
-+ third_party/tensorrt/BUILD.tpl                     |  1 +
-+ third_party/tensorrt/tensorrt_configure.bzl        |  2 +
-+ 5 files changed, 35 insertions(+), 19 deletions(-)
++ .../core/platform/default/build_config/BUILD  |  4 ++
++ tensorflow/stream_executor/cuda/BUILD         |  2 +-
++ tensorflow/tensorflow.bzl                     |  4 +-
++ ...m_google_absl_fix_mac_and_nvcc_build.patch | 45 +++++++++++--------
++ third_party/gpus/cuda/BUILD.tpl               |  8 ++++
++ third_party/gpus/cuda_configure.bzl           | 12 +++++
++ third_party/gpus/find_cuda_config.py          |  2 +
++ third_party/tensorrt/BUILD.tpl                |  1 +
++ third_party/tensorrt/tensorrt_configure.bzl   |  2 +
++ 9 files changed, 60 insertions(+), 20 deletions(-)
 +
++diff --git a/tensorflow/core/platform/default/build_config/BUILD b/tensorflow/core/platform/default/build_config/BUILD
++index 83eadf2c460..b17542c0fc6 100644
++--- a/tensorflow/core/platform/default/build_config/BUILD
+++++ b/tensorflow/core/platform/default/build_config/BUILD
++@@ -199,8 +199,12 @@ cc_library(
++             "-Wl,-rpath,../local_config_cuda/cuda/extras/CUPTI/lib",
++         ],
++         "//conditions:default": [
```

Comment:
This is nasty, but necessary. Thanks for working this out! Once Anaconda moves up to a centos7 base for x86 we'll be able to remove this stuff.